ADAL/src/ADAuthenticationContext+Internal.m

```diff
@@ -195,6 +196,10 @@ + (BOOL)canHandleResponse:(NSURL *)response
     
     if (!resumeDictionary) MSID_LOG_INFO(nil, @"No resume dictionary found.");
     
+    NSString *redirectUri = [resumeDictionary objectForKey:@"redirect_uri"];
+    BOOL notMatchRedirectUri = redirectUri && ![response.absoluteString.lowercaseString hasPrefix:redirectUri.lowercaseString];
```

Comment:
nit: can we do matchRedirectUri?  this will get into double contracts, and i hate decrypting that :p

Reply:
addressed :)